UI/window-basic-filters.cpp

```diff
@@ -121,13 +122,20 @@ OBSBasicFilters::OBSBasicFilters(QWidget *parent, OBSSource source_)
 	};
 
 	enum obs_source_type type = obs_source_get_type(source);
-	uint32_t caps = obs_source_get_output_flags(source);
 	bool drawable_type = type == OBS_SOURCE_TYPE_INPUT ||
 		type == OBS_SOURCE_TYPE_SCENE;
 
-	if (drawable_type && (caps & OBS_SOURCE_VIDEO) != 0)
-		connect(ui->preview, &OBSQTDisplay::DisplayCreated,
+	drawablePreview = (caps & OBS_SOURCE_VIDEO) != 0;
+	if (drawablePreview) {
+		ui->rightLayout->setContentsMargins(0, 0, 0, 0);
+		ui->preview->show();
+		if (drawable_type)
+			connect(ui->preview, &OBSQTDisplay::DisplayCreated,
 				addDrawCallback);
```

Comment:
Another nitpick: to keep indentation consistent between the `connect` call and `addDrawCallback`, you should probably increase indentation here